.github/workflows/build-and-push.yml

```diff
@@ -0,0 +1,55 @@
+name: Publish image to GitHub Container Registry
+
+on:
+  workflow_run:
+    workflows: [Run E2E Tests]
+    types:
+      - completed
+
+jobs:
+  on-success:
+    runs-on: ubuntu-22.04
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
+    steps:
+      - name: Checkout project
+        uses: actions/checkout@v3
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Docker Metadata action
+        id: meta
+        uses: docker/metadata-action@v5
+        env:
+          DOCKER_METADATA_PR_HEAD_SHA: "true"
+        with:
+          images: |
+            ghcr.io/buildwithgrove/path
+          tags: |
+            type=semver,pattern={{version}}
+            type=semver,pattern={{major}}.{{minor}}
+            type=ref,event=tag,suffix=-rc
+            type=sha,suffix=-rc
+            type=sha,format=long,suffix=-rc
+
+      ## TODO: Is the GH token in place in the repo?
```

Comment:
This seems like a `TODO_IN_THIS_PR`.

See https://github.com/pokt-network/poktroll/blob/ecf74ced63cc920181827ebfb04bf03853fff210/.github/workflows/reviewdog.yml#L12

Reply:
I've solved this doubt thanks to @okdas